addons/rook/1.10.6/install.sh

```diff
@@ -306,21 +282,35 @@ function rook_cluster_deploy_upgrade() {
         bail "New Ceph version failed to deploy"
     fi
 
+    logStep "Applying changes"
     rook_patch_insecure_clients
+    kubectl -n rook-ceph delete --ignore-not-found priorityclass rook-critical
+    rook_cluster_deploy_upgrade_flexvolumes_to_csi
```

Comment:
The above steps were preformed after we check the Ceph status which could result in problems
We should only check the Ceph status after perform all operations to ensure that all is fine.